components/DashboardLayout/Sidebar.tsx

```diff
@@ -83,13 +85,42 @@ const Sidebar: React.FC = () => {
         {SidebarsByRole[session.sessionType]}
       </div>
       <div className="mb-12 px-6">
-        <div className="flex items-center">
-          <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
-          <div>
-            <p className="font-semibold">{session.user?.name}</p>
-            <p className="text-unselected">
-              {UserRoleLabel[session.sessionType]}
-            </p>
+        <div>
+          <div className="flex items-center">
+            <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
+            <div>
+              <p className="font-semibold">{session.user?.name}</p>
+              <p className="text-unselected">
+                {UserRoleLabel[session.sessionType]}
+              </p>
+            </div>
+            <div
+              className="absolute h-24"
+              style={{ width: showHoverMenu ? "22rem" : "22rem" }}
+              onMouseEnter={() => {
+                setShowHoverMenu(true);
+              }}
+              onMouseLeave={() => {
+                setShowHoverMenu(false);
+              }}
+            >
+              {showHoverMenu && (
+                <div className="absolute ml-56 w-32 h-24 rounded-md shadow-lg bg-white border">
+                  <div className="h-14 pt-3 pb-3 hover:bg-hover hover:font-semibold cursor-pointer">
+                    <Link href={`/admin/user/${session.user?.id}`}>
```

Comment:
We want non-admin users to also be able to see their own profile and edit their profile page, but not be able to "close account" their own account. This could be done by moving the page out of admin and then adding a check to the session user to see if a "close account" should be shown (should only be shown for admin).

Reply:
fixed !